### ISSUE TITLE
Add accelerometer support

### ISSUE THREAD

**Issue by msurguy** [drewwilson/atvImg#2]
Maybe by using device accelerometer this could be more intuitive to use on mobile. Something like is implemented at http://www.inserthtml.com/demo/accelerometer/index-2.php would be great!

**Comment by hamishjacksonmee** [comment]
Agreed. Users aren't likely to touch and drag images unless prompted too. Parallax.js uses the accelerometer nicely. Awesome plugin though!

**Comment by adamschwartz** [comment]
:+1:
